planner/core/integration_test.go

```diff
@@ -4005,7 +4005,7 @@ func TestIssue29663(t *testing.T) {
 	tk.MustExec("insert into t2 values(1, 3), (1,4),(2,5),(2,6)")
 
 	tk.MustQuery("explain select one.a from t1 one order by (select two.d from t2 two where two.c = one.b)").Check(testkit.Rows(
-		"Projection_16 10000.00 root  test.t1.a",
+		"Projection_16 10000.00 root  test.t1.a->test.t1.a",
```

Comment:
This kind of information seems useless.
Since there has been user complaining the explain info of `Projection` is too long, I tend to be a bit conservative and avoid useless content here.